kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/AccordionContainer.vue

```diff
@@ -24,21 +57,34 @@
 
 <script>
 
+  import { enhancedQuizManagementStrings } from 'kolibri-common/strings/enhancedQuizManagementStrings';
+
   export default {
     name: 'AccordionContainer',
-    data() {
+    setup() {
+      const { expandAll$, collapseAll$ } = enhancedQuizManagementStrings;
       return {
-        expandedItemIds: [],
+        expandAll$,
+        collapseAll$,
       };
     },
-    watch: {
-      expandedItemIds() {
-        this.$emit('toggled', this.expandedItemIds);
+    props: {
+      items: {
+        type: Array,
+        required: true,
+        function(value) {
+          return value.every(item => typeof item === 'object' && 'id' in item);
```

Comment:
TIL 🤯 `'id' in { id: 1 }` => `true` 

So many times I've written `Object.keys(theObj).includes(theKey)` and I could have just used `in`